rust-toolchain.toml

```
@@ -0,0 +1,3 @@
+[toolchain]
+profile = "default" # include rustfmt, clippy
+channel = "1.80.0"
```

```suggestion
channel = "1.82.0"
```

While we are at it should we go to 1.82 as it is current latest?

---

addressed: https://github.com/FuelLabs/forc-wallet/pull/220/commits/6b82bc411fae728da93b8f11d3ee393c26625eb3